stabilizer/src/main/java/com/hazelcast/stabilizer/worker/tasks/SingleOperationAbstractWorkerTask.java

```diff
@@ -0,0 +1,33 @@
+package com.hazelcast.stabilizer.worker.tasks;
+
+/**
+ * Single operation version of {@link AbstractWorkerTask}.
+ * <p/>
+ * There is no {@link com.hazelcast.stabilizer.worker.selector.OperationSelector} used in this worker.
+ */
+public abstract class SingleOperationAbstractWorkerTask extends AbstractWorkerTask {
+
+    public SingleOperationAbstractWorkerTask() {
+        super();
+    }
+
+    @Override
+    public void run() {
+        beforeRun();
+
+        while (!testContext.isStopped()) {
+            doRun();
+
+            increaseIteration();
+        }
+        operationCount.addAndGet(iteration % performanceUpdateFrequency);
+
+        afterRun();
+    }
+
+    @Override
```

Comment:
You are missing context. This is just the fake implementation of the `AbstractWorkerTask` method. It will not be used. I added an exception to make this more clear (and in case someone tries to call it).